### ISSUE TITLE
scroll arrows over scrollbar groove?

### ISSUE THREAD

**Issue by sixsixfive** [tsujan/Kvantum#76]
![unbenannt3](https://cloud.githubusercontent.com/assets/3780774/14065174/ea94bc0a-f41f-11e5-80b8-af2adf574df1.png)

i guess its not possible to make them look like the gimp scrollbars in the screenshot?

**Comment by tsujan** [comment]
The screenshot is wrong but I'm looking at GIMP's scrollbars with Adwaita and see nothing special that cannot be made by Kvantum.

**Comment by sixsixfive** [comment]
so how can i get this?

![get](https://cloud.githubusercontent.com/assets/3780774/14065287/ff4b7acc-f423-11e5-9fb9-700a047edc3e.png)

**Comment by tsujan** [comment]
You could set zero top and bottom frames for ScrollbarGroove and use appropriate up and down shapes for Scrollbar indicators consisting of boxes with arrows inside them. This is a rough sketch for the "up" indicator:
![up](https://cloud.githubusercontent.com/assets/981076/14065347/34206552-f43b-11e5-87bc-1b770f3e02ce.png)

As is explained in the doc, the horizontal scrollbar is automatically made out of the vertical.

**Comment by tsujan** [comment]
Wait! It seems it doesn't work that way.

**Comment by tsujan** [comment]
I'm very busy right now but I'll add such a possibility as soon as I have time. It's a good idea that I hadn't considered before because I didn't use it in my themes.

I'll tell you when it's implemented.

**Comment by tsujan** [comment]
OK, I'm back. There was no need to any change for this. As an example, I used these lines in your theme Glare:
````
[Scrollbar]
indicator.element=scrollArrow
frame=false
frame.element=scrollbar
interior=false
interior.element=scrollbar
indicator.size=16

[ScrollbarGroove]
inherits=Scrollbar
frame=true
interior=true
frame.top=0
frame.bottom=0
frame.left=2
frame.right=2
````
And added `scrollArrow` up and down objects in all states to it, as sketched above. The result was what you wanted:

![scrollbar](https://cloud.githubusercontent.com/assets/981076/14066986/e09fbc64-f46e-11e5-94a6-700aa6f731f3.png)

`indicator.size` should be 16 because `scroll_width` was 16 in Glare. I made `scrollArrow` objects (one was enough -- the others could be made by duplication or cloning) as 16x16px rectangles with 2px borders because, in the above example, I chose 2px right and left frames for ScrollbarGroove.

Conclusion: Kvantum is so flexible as it is :)
